kde-misc/playbar2/playbar2-2.5.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils kde5-functions
+
+DESCRIPTION="MPRIS2 client, written in QML for Plasma 5"
+HOMEPAGE="https://github.com/audoban/PlayBar2"
+SRC_URI="https://github.com/audoban/PlayBar2/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE=""
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="$(add_frameworks_dep extra-cmake-modules)
+	$(add_frameworks_dep kconfigwidgets)
+	$(add_frameworks_dep kdeclarative)
```

Comment:
no linking with kdeclarative found, is it just a runtime-dep?

Reply:
I confused it with qtdeclarative. That is given by the find_package. However, dynlink-scanner does not find this dep. Howto handle such situation? Is it an obsolete include in the CMakeList?

Reply:
OK, I will keep the qtdeclerative anyway since find_package would fail otherwise (even if it is not detected by dynlink-scanner).